src/app/ubs/ubs-admin/components/add-violations/add-violations.component.html

```diff
@@ -5,13 +5,13 @@
   <div *ngIf="!viewMode" class="row mb-4">
     <h3>{{ 'add-violation-modal.add-violation' | translate }}</h3>
   </div>
-  <div *ngIf="viewMode" class="row mb-4 view-header">
+  <div *ngIf="viewMode" class="row mb-3 view-header">
     <h3>{{ 'add-violation-modal.violation' | translate }}</h3>
     <span>{{ 'add-violation-modal.order' | translate }} №{{ orderId }}</span>
   </div>
   <div class="row justify-content-end edit-violation">
+    <mat-icon class="mat-icon">edit</mat-icon>
```

Comment:
We should see this icon in Add violation
Actual result:
![image](https://user-images.githubusercontent.com/104850911/196130105-dfd21ab5-5e5c-4687-8a4a-68fcbe54942e.png)
Expected result: 
![image](https://user-images.githubusercontent.com/104850911/196130245-ecdc7edc-e8db-4894-99d1-93cd6455b59f.png)

Reply:
Yes, you're right.

Reply:
Fixed this behaviour. Added new commtit.